### ISSUE TITLE
tensorflow albert optimization Error

### ISSUE THREAD

**Issue by jianqianzhou** [onnx/onnx#3308]
`
output_model_path="segment.onnx"
opt_options = BertOptimizationOptions('bert_tf')
opt_options.enable_embed_layer_norm = False

opt_model = optimizer.optimize_model(
output_model_path,
'bert_keras',
num_heads=12,
hidden_size=768,
optimization_options=opt_options)
opt_model.save_model_to_file('segment.opt.onnx')`

`opt_model_path="segment.opt.onnx"
session = onnxruntime.InferenceSession(opt_model_path, sess_options, providers=['CPUExecutionProvider'])
import numpy as np
batch_size=1
inputs_onnx = {k_+":0": np.repeat(v_, batch_size, axis=0) for k_, v_ in model_inputs.items()}
logits = session.run(None, inputs_onnx)`

optimized tensorflow albert onnx,but run this opt.onnx an error is occured as follows
onnxruntime.capi.onnxruntime_pybind11_state.Fail: [ONNXRuntimeError] : 1 : FAIL : Load model from segment.opt.onnx failed:This is an invalid model. Node (StatefulPartitionedCall/tf_albert_model/albert/encoder/albert_layer_groups_.0/albert_layers._0/attention/add) input arg (StatefulPartitionedCall/tf_albert_model/albert/mul:0) does not have type information set by parent node.

**Comment by askhade** [comment]
@jianqianzhou please open this issue in onnxruntime repo: https://github.com/microsoft/onnxruntime/issues
